components/nimbus/ios/scripts/nimbus-fml.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
```

Comment:
This file is taken from Firefox iOS' `./bin/nimbus-fml.sh`, and lightly modified for shell check reasons.